packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -272,6 +225,17 @@ class MemoryBodyState extends State<MemoryBody> {
             minIncludeTextWidth: 1100,
           ),
         ),
+        const SizedBox(width: 16.0),
+        OutlineButton(
+          key: MemoryScreen.exportButtonKey,
+          onPressed:
+              controller.offline ? null : controller.memoryLog.exportMemory,
```

Comment:
when the value of `controller.offline` changes, how is the UI getting updated? Consider making offline a ValueNotifier and having widgets listen for changes to the value with ValueListenableBuilder